AppSettingsManagerBff/AppSettingsManagerBff.Infrastructure/ApiRepositories/HttpSettingsRepository.cs

```diff
@@ -48,4 +48,62 @@ public async Task<ApiSetting> CreateSetting(CreateSettingRequest request)
         // "??" means "if null then"
         return setting ?? throw new HttpRequestException();
     }
-}
+
+    public async Task<ApiSetting> GetSetting([FromRoute][Required] string settingId, [FromRoute][Required] int version)
+    {
+        // Send request
+        var response = await _httpClient.GetAsync($"{settingId}/{version}");
+        response.EnsureSuccessStatusCode();
+
+        // Extract response content as json string
+        var jsonResponse = await response.Content.ReadAsStringAsync();
+
+        // Deserialize json string
+        var setting = JsonSerializer.Deserialize<ApiSetting>(jsonResponse, _jsonSerializerOptions);
+
+        // "??" means "if null then"
+        return setting ?? throw new HttpRequestException();
+    }
+
+    public async Task<ApiSetting> UpdateSetting([FromRoute][Required] UpdateSettingRequest request)
+    {
+        // Serialize request object to string
+        var serializedRequest = JsonSerializer.Serialize(request);
+
+        // Wrap request object in StringContent object
+        var content = new StringContent(
+            serializedRequest,
+            System.Text.Encoding.UTF8,
+            "application/json"
+        );
+
+        // Send request
+        var response = await _httpClient.PutAsync($"{request.SettingId}/{request.Version}", content);
+        response.EnsureSuccessStatusCode();
+
+        // Extract response content as json string
+        var jsonResponse = await response.Content.ReadAsStringAsync();
+
+        // Deserialize json string
+        var setting = JsonSerializer.Deserialize<ApiSetting>(jsonResponse, _jsonSerializerOptions);
+
+        // "??" means "if null then"
+        return setting ?? throw new HttpRequestException();
+    }
+
+    public async Task<ApiSetting> DeleteSetting([FromRoute][Required] string settingId)
```

Comment:
```suggestion
    public async Task<IEnumberable<ApiSetting>> DeleteSetting([FromRoute][Required] string settingId)
```

This method returns a list of settings
https://github.com/WSU-4110/AppSettingsManager/blob/48e5cecdcb53e26c8dffd440d22b28bdf86f42e1/AppSettingsManagerAPI/AppSettingsManagerApi/Controllers/SettingsController.cs#L69

Reply:
So I get the change you would like for me to commit, but can you explain the second comment you made for [Line 69](https://github.com/WSU-4110/AppSettingsManager/blob/48e5cecdcb53e26c8dffd440d22b28bdf86f42e1/AppSettingsManagerAPI/AppSettingsManagerApi/Controllers/SettingsController.cs#L69).

Reply:
I was just pointing you to the method in the API that you're calling

Reply:
Ah, ok, but  do you need me to redefine it?? Or is my definition ok for now??

Reply:
This is good now. See my other comments